app/models/rules_factory.rb

```diff
@@ -4,10 +4,18 @@ def initialize(auction)
   end
 
   def create
+    eligibility = if auction.small_business?
```

Comment:
might feel like overkill here, but it seems like it shouldn't be the responsibility of `RulesFactory` to know about eligibility -- thoughts on creating a factory class for `Eligiblity` similar to `RulesFactory` that we can call here?